src/views/BusinessCase/AlternativeSolution/index.tsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import { Field, FormikProps } from 'formik';
+import Label from 'components/shared/Label';
+import HelpText from 'components/shared/HelpText';
+import TextField from 'components/shared/TextField';
+import TextAreaField from 'components/shared/TextAreaField';
+import FieldGroup from 'components/shared/FieldGroup';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import Button from 'components/shared/Button';
+import EstimatedLifecycleCost from 'components/EstimatedLifecycleCost';
+import { defaultProposedSolution } from 'views/BusinessCase';
+import {
+  BusinessCaseModel,
+  ProposedBusinessCaseSolution
+} from 'types/businessCase';
+
+type AlternativeSolutionProps = {
+  formikProps: FormikProps<BusinessCaseModel>;
+  altLetter: string;
+  handleToggleAlternative?: () => void;
+};
+
+const AlternativeSolution = ({
+  formikProps,
+  altLetter,
+  handleToggleAlternative
+}: AlternativeSolutionProps) => {
+  const { values } = formikProps;
+  const altLabel = `Alternative ${altLetter}`;
+  const altId = `alternative${altLetter}`;
+  const altValues = ((letter: string): ProposedBusinessCaseSolution => {
```

Comment:
This is an immediately invoked function expression (IIFE) that will return the Formik values based on the letter type.